waffle/admin.py

```diff
@@ -41,14 +43,44 @@ def delete_individually(ma, request, qs):
 delete_individually.short_description = _('Delete selected')
 
 
+class InformativeManyToManyRawIdWidget(ManyToManyRawIdWidget):
+    """Widget for ManyToManyField to Users.
+
+    Will display the names of the users in a parenthesised list after the
+    input field. This widget works with all models that have a "name" field.
+    """
+    def label_and_url_for_value(self, values):
+        names = []
+        key = self.rel.get_related_field().name
+        for value in values:
+            try:
+                name = self.rel.model._default_manager \
+                    .using(self.db) \
+                    .get(**{key: value})
+                names.append(escape(str(name)))
+            except self.rel.model.DoesNotExist:
+                names.append('<missing>')
+        return "(" + ", ".join(names) + ")", ""
+
+
 class FlagAdmin(BaseAdmin):
     actions = [enable_for_all, disable_for_all, delete_individually]
     list_display = ('name', 'note', 'everyone', 'percent', 'superusers',
                     'staff', 'authenticated', 'languages')
     list_filter = ('everyone', 'superusers', 'staff', 'authenticated')
-    raw_id_fields = ('users', 'groups')
+    raw_id_fields = ('users', )
     ordering = ('-id',)
 
+    def formfield_for_dbfield(self, db_field, **kwargs):
+        if db_field.name == 'users':
+            kwargs.pop('request', None)
+            kwargs['widget'] = \
+                InformativeManyToManyRawIdWidget(db_field.remote_field,
+                                                 self.admin_site,
+                                                 using=kwargs.get("using"))
+            return db_field.formfield(**kwargs)
+        return super().formfield_for_dbfield(db_field, **kwargs)
```

Comment:
`return super(FlagAdmin, self).formfield_for_dbfield(db_field, **kwargs)` for python 2 compatibility